mani_skill/envs/scene.py

```diff
@@ -129,13 +129,15 @@ def create_physical_material(
     ):
         return physx.PhysxMaterial(static_friction, dynamic_friction, restitution)
 
-    def remove_actor(self, actor):
+    def remove_actor(self, actor : Actor):
```

Comment:
python format is wrong here. Please follow the CONTRIBUTING.md instructions for setting up pre-commit which comes with black and isort for formatting.